metabase/metabase
# FE - e2e - Cover dashboard drills with tests

Closes #47219

Epic: #46519
Testing plan: #46520

### Description

:arrow_right:  Use **hide whitespace** option when reviewing this PR ([image](https://github.com/user-attachments/assets/5c090510-72e5-419c-8b68-08780f91e8cd)).

PR adds tests for applying filters for various query stages in dashboards and drilling via dashcard header with the filters applied. It also adds missing FE-side parts for this very feature.

Many tests are skipped, many assertions are commented out. Every such case should have a link to a related GH issue.

Tests for combination of unit of time parameter + non-last-stage filter will come separately (#48445).



### How to verify

- Check if skipped tests and commented out assertions make sense conceptually.
- 2 CI failures are expected:
    - `markdown-link-check`
    - `e2e-tests-dashboard-filters-ee` 
        - I had to introduce a new group because tests were timing out due to 45min limit 
        - this group should succeed in #48610
